src/vue-pdf-embed.vue

```diff
@@ -294,31 +298,35 @@ export default {
 
         await Promise.all(
           this.pageNums.map(async (pageNum, i) => {
+            const pageRotation = this.rotation + page.rotate
             const page = await this.document.getPage(pageNum)
```

Comment:
`pageRotation` depends on `page`, thus must be defined after 

```suggestion
            const page = await this.document.getPage(pageNum)
            const pageRotation = this.rotation + page.rotate
```